tests/resolvers/Mutation/createOrganization.spec.ts

```diff
@@ -79,13 +82,58 @@ describe("resolvers -> Mutation -> createOrganization", () => {
     }
   });
 
+  it(`throws Not Authorised Error if user is not a super admin`, async () => {
```

Comment:
Here's you are checking for the `isSuperAdmin` edge case. In `acceptAdmin.spec.ts` you didn't. Choose one approach.